data/ecosystem/vendors.yaml

```diff
@@ -23,7 +23,7 @@
   oss: false
   commercial: true
 - name: AWS
-  nativeOTLP: false
+  nativeOTLP: true
   url: https://aws-otel.github.io
   contact:
```

Comment:
Any chance you could add a contact as well? Just in case we need to reach out. Thanks!

Reply:
Thank you! I added the contact.